packages/jobs/package.json

```diff
@@ -27,7 +27,7 @@
         "@trpc/client": "^10.44.1",
         "@trpc/server": "^10.44.1",
         "@types/fs-extra": "^11.0.1",
-        "dd-trace": "4.20.0",
+        "dd-trace": "5.2.0",
```

Comment:
Upgrading all the packages because they changed the types (the main breaking change was the drop of Node16)